### ISSUE TITLE
error: net::ERR_PROXY_CONNECTION_FAILED.

### ISSUE THREAD

**Issue by sxarthur** [kakaroto/SWProxy#138]
Since AUG 6 (when i first installed) when trying to get in-game, I get the error: net::ERR_PROXY_CONNECTION_FAILED.
It read the files, It loads but gives the error when i press "touch to play".
everything else seems to be working. game runs normally without the proxy.

**Comment by Aebsol** [comment]
Try running the proxy in admin mode.

**Comment by sxarthur** [comment]
I've tried on 2 diferent pcs, and on 3 diferent devices, on admin mode.

still having the error... maybe is my router? my router is technicolor tc7300.b 
any tips/ideas?

**Comment by Viper2k4** [comment]
"net::ERR_PROXY_CONNECTION_FAILED" on Android normally means that the device cant reach the proxy it self. Are u sure u use the correct port and ip? U disable the Firewall on ur PC?

**Comment by sxarthur** [comment]
Yes, i double checked everytime. firewall was ON, but turned it off and tried again, but still get the error

**Comment by AizenSousuke** [comment]
Still working for me. Make sure the ip is correct on your phone.

**Comment by sxarthur** [comment]
like i said, i double checked everytime, tried again today, same thing

**Comment by jake-griffin** [comment]
Are the computer and phone connected to the same wireless network?

**Comment by Fuusetsu** [comment]
Same issue at my side I do not know why :/

**Comment by Viper2k4** [comment]
i dont wannt blame u. But i would say this is 90% a User Error :/

I go "error: net::ERR_PROXY_CONNECTION_FAILED" if my Device cant connect to the Proxy. Because of Filewall or anything else. The Problem here ist, its hard to find the error remote.

It is simple as it is:

- start SWProxy 
- click start
- read the IP and Port shown in the GUI
- enter the WiFi Settings in ur Device
- Switch Proxy settings to "Manuell"
- Enter the IP and the Port and save it (on iOS only co back)
- Start/Restart Summoners War

If this not Work u could check the log from SWProxy. If u dont find anything in here u should try to disable the Firewall (windows use its own by default, maybe ur router also)

**Comment by Fuusetsu** [comment]
It is ok now, it seems to be the router, because I have tried with my phone as a router and now it is ok.

**Comment by rm00** [comment]
@Fuusetsu I had exactly the same problem about a month ago, I solved forgetting the network.

**Comment by sxarthur** [comment]
i created port rules, fowarding and stuff on windows firewall.  also forgot the network and tried again.
but i guess i found out whats happening, my ISP is blocking this port.  how can i really check if a port is blocked? are there any alternative ports i could run the swproxy?

**Comment by jake-griffin** [comment]
I run the proxy (on a Mac) with "python SWProxy.py -p 8888". I have issues serving it up on port 8080, but 8888 works just fine for me. You can really use any port number you want, from 1025 up to 65535 (1-1024 are "reserved" for "privileged services"). If 8080 isn't working for you, try entering a random port number (1025 to 65535) and see if that works.

**Comment by DelilahSucks** [comment]
Hey I have done all the things mentioned above to try and fix the connection issue. I have changed the ports numbers, ensured I have the correct IP address, I have restarted the device and wifi multiple times, yet I am unable to get the Proxy working. i have also turned off the Firewall as well. Please Help.

**Comment by ffhighwind** [comment]
I had this same error and finally got it working. I did this by running the Istern fork from a laptop. It wasn't working when I ran the program from my PC that is wired directly to the router.

Windows 7 + Android phone.

**Comment by reapermech** [comment]
just started getting this issue today , my ports match , ip match devices are on the same network, firewall was disabled for testing and the error still persists .
have had no issues prior to this last patch , could there be something else that is causing this issue for android users ?

**Comment by AizenSousuke** [comment]
i'm having the same issue as OP. It used to work fine. I think the way my router is configured has some problem with it - I can't even FTP to my phone from my pc using the connection. I solved it by making a virtual wifi and connected my phone to it. It works without the darn router (I have no access to router). Can you try ftping to your device and see if it works or not. Or just reset the router and try again.

**Comment by hsds312** [comment]
Try clicking the IP address box and select the secondary address if there are any listed there, then enter that as the proxy hostname. Had the same issue even though I entered all the information correctly and disabled my firewall, this was what solved my issue.

**Comment by manikandanbaskarans** [comment]
same issue for me in my mobile network. I use a burp laptop, intercept to my mobile1. I connected both devices to mobile2's network.pls, give me a clarification on "is it possible on the mobile network?"

**Comment by manikandanbaskarans** [comment]
@AizenSousuke @jake-griffin @rm00 @Fuusetsu @ffhighwind                     same issue for me in my mobile network. I use a burp laptop, intercept to my mobile1. I connected both devices to mobile2's network.pls, give me a clarification on "is it possible on the mobile network?"
